android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java

```diff
@@ -1602,6 +1602,16 @@ private void invokeCreateInstance(@NonNull WXSDKInstance instance, Script templa
           }
           return;
         } else {
+          if (instance.getRenderStrategy() != WXRenderStrategy.DATA_RENDER_BINARY && instance.getRenderStrategy() != WXRenderStrategy.DATA_RENDER) {
```

Comment:
The `if` in 1592 already considered DATA_RENDER or DATA_RENDE_BINARY.

No need to do it twice.